assets/questions/reverseWords.js

```diff
@@ -93,6 +97,14 @@ globalData.questions['reverseWords'] = {  // eslint-disable-line dot-notation
           "last reversed word."
         ].join('')
       ]
+    }, {
+      buggyFunctionName: 'AuxiliaryCode.reverseString',
+      messages: [
+        [
+          "Please re-read the question. You need to reverse each word ",
```

Comment:
Good catch, thanks -- I wanted to highlight that, in general, reading the question correctly is an important thing to do, but I agree that the way it's worded here comes off wrong. I've changed it to the following; is it OK?

> Whoops! It looks like you're reversing the entire string, but you should only be reversing each word in turn, not their ordering. Please check the given example, and try again.